src/fr/poseidonscans/src/eu/kanade/tachiyomi/extension/fr/poseidonscans/PoseidonScansDto.kt

```diff
@@ -0,0 +1,49 @@
+package eu.kanade.tachiyomi.extension.fr.poseidonscans
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class LatestApiManga(
```

Comment:
Don't use `data class` unless you need it. The most noticable difference is that `data class` stringifies to something easy to debug with.
```suggestion
class LatestApiManga(
```